src/components/Setup/ProtocolCard.js

```diff
@@ -1,25 +1,126 @@
+      type: 'Error',
+      title: 'Obsolete Protocol Schema',
+      canCancel: false,
+      message: (
+        <React.Fragment>
+          <p>
+            This protocol uses an obsolete version of the protocol file format, or
+            &quot;schema&quot;.
+          </p>
+          <p>
+            The version of the schema used by this protocol is incompatible with this version of
+            Network Canvas. You may still export interview data that has already been collected,
+            but you may not start additional interviews.
+          </p>
+          <p>
+            If you require the ability to start interviews, you can either (1) install an updated
+            version of this protocol that uses the latest schema, or (2) downgrade your version
+            of Network Canvas to a version that supports this protocol schema version.
+          </p>
+          <p>
+            For documentation on this issue, please see our documentation site.
+          </p>
+        </React.Fragment>
+      ),
+    });
+  };
+
+  const modifierClasses = cx(
+    'protocol-card',
+    { 'protocol-card--warning': !isObsoleteProtocol() && isOutdatedProtocol() },
+    { 'protocol-card--error': isObsoleteProtocol() },
+  );
+
+  const renderCardIcon = () => {
+    const iconMarkup = isOutdatedProtocol() ?
+      (
+        <div className="protocol-card__warning" onClick={handleSchemaOutdatedInfo}>
+          <Icon name="warning" />
+        </div>) :
+      (
+        <div className="protocol-card__error" onClick={handleSchemaObsoleteInfo}>
+          <Icon name="error" />
+        </div>
+      );
+
+    return iconMarkup;
   };
 
   return (
-    <div className="protocol-card" onClick={() => selectProtocol(protocol)}>
-      <div className="protocol-card__delete" onClick={handleDelete}>
+    <div className={modifierClasses}>
+      <div className="protocol-card__delete" onClick={handleDeleteProtocol}>
         <Icon name="delete" />
       </div>
+      {
+        isOutdatedProtocol() || isObsoleteProtocol() ?
+          renderCardIcon() : ''
+      }
       <h2 className="protocol-card__name">{protocol.name}</h2>
-      <div className="protocol-card__icon-wrapper">
-        <Icon className="protocol-card__icon" name="protocol-card" />
+      <p className="protocol-card__description">
+        { protocol.description ?
+          protocol.description : (<em>No protocol description.</em>)}
+      </p>
+      <div
+        className={isObsoleteProtocol() ? ('start-button start-button--disabled') : ('start-button')}
+        onClick={() => selectProtocol(protocol)}
+      >
+        <Icon className="start-button__protocol-icon" name="protocol-card" />
+        <div className="start-button__text">Start new interview</div>
+        <Icon className="start-button__arrow" name="chevron-right" />
       </div>
-      { protocol.description ? (<p className="protocol-card__description">{protocol.description}</p>) : ''}
     </div>
   );
 };
```

Comment:
Consider making this a class and defining some of these staetments as constants outside the render method.